pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
> BIT64 refers to 64 bit build -- x64 or arm64.
So, what is the problem with proposed fix at line 203 ?
Second;
The problem is when __int64 is long, _Is_integral has duplicate definitions. See https://github.com/Microsoft/ChakraCore/pull/157/files#diff-050fe98eebc72bf7b89245109bb89320L311 and line 317